src/components/formik/fieldComponents/CheckRadioFieldComponent.tsx

```diff
@@ -1,7 +1,9 @@
+/* eslint-disable jsx-a11y/label-has-for */
+/* eslint-disable jsx-a11y/label-has-associated-control */
```

Comment:
Similar to above comment, I would prefer if this is disabled just for the next line. If it's not possible to fix the lint errors that way though then we can leave it

Reply:
I would like to leave this for ow.  Can you add an issue if you want?